ogr/services/github.py

```diff
@@ -269,6 +270,24 @@ def pr_create(
         )
         return self._pr_from_github_object(created_pr)
 
+    def update_pr_info(self, pr_id: int, title: str, description: str):
+        """
+        Update pull-request information.
+
+        :param pr_id: int The ID of the pull request
+        :param title: str The title of the pull request
+        :param description str The description of the pull request
+        :return: PullRequest
+        """
+        pr = self.github_repo.get_pull(number=pr_id)
+        try:
+            pr.edit(title=title, body=description)
+            logger.info(f"PR updated: {pr.url}")
+            return self._pr_from_github_object(pr)
+        except Exception as ex:
+            logger.error("there was an error while update a PR: %r", ex)
+            raise
```

Comment:
It would be nice to raise some ogr-specific exception -- you can create a new one for Github like we have `PagureAPIException` in ./ogr/exceptions.py.